gcs/src/dashboard.jsx

```diff
@@ -419,7 +435,7 @@ export default function Dashboard() {
             }}
             className='h-full'
           >
-            <div className='flex flex-col p-2 h-full gap-2'>
+            <div className='flex flex-col p-1 h-full gap-2 overflow-x-hidden overflow-y-auto'>
```

Comment:
overflow hidden could create issues for joe's changes, is this needed? #306

Reply:
It adds scrollbar and stuff, idk what Joe's doing so not sure, we can merge and he can change it afterwards if he needs to?

Reply:
Sure